ee/app/livechat-enterprise/server/lib/Helper.js

```diff
@@ -144,7 +144,7 @@ export const setPredictedVisitorAbandonmentTime = (room) => {
 };
 
 export const updatePredictedVisitorAbandonment = () => {
-	if (settings.get('Livechat_auto_close_abandoned_rooms')) {
+	if (settings.get('Livechat_abandoned_rooms_action') && (settings.get('Livechat_abandoned_rooms_action') === 'none')) {
```

Comment:
Are you sure? I didn't get it.

Reply:
Sorry about this, the logic here was wrong. I've fixed it to this now
```
if (!settings.get('Livechat_abandoned_rooms_action') || (settings.get('Livechat_abandoned_rooms_action') === 'none')) {
   LivechatRooms.unsetPredictedVisitorAbandonment();
} else {
   LivechatRooms.findLivechat({ open: true }).forEach((room) => setPredictedVisitorAbandonmentTime(room));
}
```